db/migrate/20160713003535_add_bill_address_and_ship_address_to_customer.rb

```diff
@@ -0,0 +1,12 @@
+class AddBillAddressAndShipAddressToCustomer < ActiveRecord::Migration
+  def change
+    add_column :customers, :bill_address_id, :integer
+    add_column :customers, :ship_address_id, :integer
+
+    add_index :customers, :bill_address_id
```

Comment:
Just thinking through. It's a one-to-one relationship. You normally don't lookup a customer by it's address. That's why I thought we don't need it. But it is possible that we add a feature later to get all customers of a region. In that case the index would be good. I'm not sure what's better. Probably just leave it as is.